source/extensions/filters/listener/http_inspector/http_inspector.h

```diff
@@ -82,6 +84,10 @@ class Filter : public Network::ListenerFilter, Logger::Loggable<Logger::Id::filt
   Network::ListenerFilterCallbacks* cb_{nullptr};
   Event::FileEventPtr file_event_;
   absl::string_view protocol_;
+  http_parser parser_;
+  static http_parser_settings settings_;
+  uint32_t read_{0};
+  const int s_req_http_end_{42};
```

Comment:
how 42 is calculated? Is it from https://github.com/nodejs/http-parser/blob/master/http_parser.c#L324? since now we're only parsing request line I think just parsing until EOL and check with OK/PAUSED is enough.

Reply:
Done